src/components/Header.tsx

```diff
@@ -5,17 +5,22 @@ import {
   handleGoogleLogout,
   handleGoogleAuthStateChange,
 } from '../api/firebase';
-import { userStore } from '../store/useUserStore';
+import { useUserStore } from '../store/useUserStore';
 import User from './User';
 import styled from 'styled-components';
 import ButtonComponent from './ui/ButtonComponent';
 import basicLogo from '../images/basicLogo.png';
 import { lightTheme } from '../css/styles.theme';
+import { useDarkModeStore } from '../store/useDarkModeStore';
 
 export default function Header() {
-  const user = userStore((state) => state.user);
-  const setUser = userStore((state) => state.actions.setUser);
-  const clearUser = userStore((state) => state.actions.clearUser);
+  const user = useUserStore((state) => state.user);
+  const setUser = useUserStore((state) => state.actions.setUser);
+  const clearUser = useUserStore((state) => state.actions.clearUser);
```

Comment:
이거 전에 리뷰 할 때 수정 제안 드렸는데 반영이 안 되어 있네요 

```suggestion
const user = useGetUser();
const { setUser, clearUser } = useUserActions();
```

store에서는 

```
export const useUserActions = () => useUserStore((state) => state.actions);
export const useGetUser = () => useUserStore((state) => state.user);

```

이렇게 바꿀 수 있을 것 같은데요 

use 훅을 제공하는 곳에서 미리 store내에 어떤 값들을 꺼내올 것인지 함수로 만들어 놓고,
사용하는 곳에서는 store내에 어떤 값을 꺼내올지 후처리를 할 필요 없이 만들어 놓은 함수만 호출하면 더 편리해요